jkube-kit/doc/src/main/asciidoc/inc/generator/_overview.adoc

```diff
@@ -1,87 +1,67 @@
+| `includes`
+| Contains one or more `include` elements with generator names which should be included. If given only this list of generators are included in this given order. The order is important because by default only the first matching generator kicks in. The generators from every active profile are included, too. However the generators listed here are moved to the front of the list, so that they are called first. Use the profile `raw` if you want to explicitly set the complete list of generators.
 
-| `<excludes>`
-| Holds one or more `<exclude>` elements with generator names to exclude. If set then all detected generators are used except the ones mentioned in this section.
+| `excludes`
+| Holds one or more `exclude` elements with generator names to exclude. If set then all detected generators are used except the ones mentioned in this section.
 
-| `<config>`
-| Configuration for all generators. Each generator support a specific set of configuration values as described in the documentation. The subelements of this section are generator names to configure. E.g. for generator `spring-boot`, the sub-element is called `<spring-boot>`. This element then holds the specific generator configuration like `<name>` for specifying the final image name. See above for an example. Configuration coming from profiles are merged into this config, but not overriding the configuration specified here.
+| `config`
+| Configuration for all generators. Each generator support a specific set of configuration values as described in the documentation. The subelements of this section are generator names to configure. E.g. for generator `spring-boot`, the sub-element is called `spring-boot`. This element then holds the specific generator configuration like `name` for specifying the final image name. See above for an example. Configuration coming from profiles are merged into this config, but not overriding the configuration specified here.
 |===
 
 Beside specifying generator configuration in the plugin's configuration it can be set directly with properties, too:
 
-.Example generator property config
-[source, sh, subs="+attributes"]
-----
-mvn -Djkube.generator.java-exec.webPort=8082
-----
+ifeval::["{plugin-type}" == "maven"]
+include::maven/_overview_generator_property.adoc[]
+endif::[]
+ifeval::["{plugin-type}" == "gradle"]
+include::gradle/_overview_generator_property.adoc[]
+endif::[]
 
 The general scheme is a prefix `jkube.generator.` followed by the unique generator name and then the generator specific key.
 
+
+ifeval::["{plugin-type}" == "maven"]
```

Comment:
I'm not sure if there's already an issue for this functionality. If there is, add the // TODO comment and reference the issue, if there isn't, please create one and reference it here.